checkov/common/output/csv.py

```diff
@@ -49,7 +49,8 @@ def __init__(self) -> None:
     def add_report(self, report: Report, git_org: str, git_repository: str) -> None:
         if report.check_type == CheckType.SCA_PACKAGE:
             for record in itertools.chain(report.failed_checks, report.passed_checks, report.skipped_checks):
-                self.add_sca_package_resources(resource=record, git_org=git_org, git_repository=git_repository)
+                if record.check_name == "SCA package scan":
```

Comment:
can we extract `"SCA package scan"` to a constant in commons should be a place for this kind of things. And then use it here and where it is actually set.

Reply:
sure! will be fixed